src/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
I think the index file.html left here doesn't make sense. You can create folders public/index.html